hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/Operation.java

```diff
@@ -652,6 +654,17 @@ public void logError(Throwable e) {
             } catch (Throwable ignored) {
                 ignore(ignored);
             }
+        } else if (e instanceof IllegalStateException) {
+            // if start is not yet completed, do not warn about IllegalStateExceptions
+            // due to cluster state being PASSIVE
+            final Level level = nodeEngine != null
+                    && nodeEngine.getClusterService().getClusterState() == ClusterState.PASSIVE
```

Comment:
I understand the intent here but the logic here will surprise all exceptions if cluster is in ACTIVE state.